app/models/setting/auth.rb

```diff
@@ -59,3 +59,4 @@ def validate_websockets_ssl_cert(record)
   end
 end
 
+require_dependency 'setting'
```

Comment:
This same issue came up separately via bug #13574 which prompted me to look into it further.  I think now that these lines are just a workaround and all along the Setting class has been only getting partially loaded, then loaded again (you can see this with the duplicate constant warnings).  Seemingly the class loading is a bit fragile to have broken twice.

#3159 should fix this more permanently so the class is only loaded once.  Once the foreman-tasks gem is available, I'll rebase this PR on top of that one and revert these changes.